core/pom.xml

```diff
@@ -185,7 +228,11 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
-
+	
+		<dependency>
```

Comment:
Not sure how kafka is utilized in this PR, but we already have spring-kafka which depends on kafka-client